extensions/dacpac/src/wizard/api/basePage.ts

```diff
@@ -122,14 +124,17 @@ export abstract class BasePage {
 	protected async getDatabaseValues(): Promise<string[]> {
 		let idx = -1;
 		let count = -1;
-		this.databaseValues = (await azdata.connection.listDatabases(this.model.server.connectionId)).map(db => {
-			count++;
-			if (this.model.database && db === this.model.database) {
-				idx = count;
-			}
+		this.databaseValues = (await azdata.connection.listDatabases(this.model.server.connectionId))
```

Comment:
does it affect/should affect any test?

Reply:
On the same lines, can we add a test around this change, if not present already? May be a separate PR, but will be good to have :).

Reply:
Doesn't affect any existing tests. Added a test to make sure system dbs get filtered out :)